openapi.license

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2018, 2024 Ericsson
+# Copyright (c) 2018, 2024, 2025 Ericsson
```

Comment:
only update the the last year (2024->2025). It's always first year, last year.

Also, changes in this file should be part of a separate commit and together with the API changes